server/pulp/server/async/worker_watcher.py

```diff
@@ -78,9 +78,6 @@ def handle_worker_heartbeat(event):
         msg = _("New worker '%(worker_name)s' discovered") % event_info
```

Comment:
Specifically, looking after itself means that it logs when it heartbeats for the first time itself.